PhysicsTools/PatAlgos/python/slimming/applyDeepBtagging_cff.py

```diff
@@ -58,19 +58,20 @@ def applyDeepBtagging( process, postfix="" ) :
        muSource = cms.InputTag('slimmedMuons'),
        elSource = cms.InputTag('slimmedElectrons'),
        rParam = 0.8,
-       jetCorrections = ('AK8PFPuppi', cms.vstring(['L1FastJet', 'L2Relative', 'L3Absolute']), 'None'),
+       jetCorrections = ('AK8PFPuppi', cms.vstring(['L2Relative', 'L3Absolute']), 'None'),
        btagDiscriminators = [
           'pfDeepDoubleBJetTags:probQ',
           'pfDeepDoubleBJetTags:probH',
        ],
-       postfix = 'SlimmedAK8DeepDoubleB'+postfix,
+#        ] + pfDeepBoostedJetTagsAll,  # uncomment it to test DeepBoostedJet
```

Comment:
@slava77 I understand that this is the line that would trigger the production of the new collection, for the erst the old one is renamed (DeepDoubleB -> DeppTags) but it is not a persistent collection

Reply:
I'm not sure I understand ```for the erst ```.

The change of the name in the transient collection names starts at L49 and the postfix above also updates other transient collection names.